dashboard/test/models/script_test.rb

```diff
@@ -722,31 +722,6 @@ def populate_cache_and_disconnect_db
     assert Script.find_by_name('ECSPD').professional_learning_course?
   end
 
-  test 'hoc?' do
```

Comment:
Removed these tests as loading lots of fixtures to just test this seemed silly and since the methods are so simple they probably don't need these tests.